style.css

```diff
@@ -1,13 +1,36 @@
 html {
-  overflow:hidden;
-  background: #fff;
+  overflow: hidden;
 }
 
-#shaper {
+.editor {
   position: absolute;
-  top: 0.5em;
-  right: 0.5em;
-  bottom: 0.5em;
-  left: 0.5em;
-  background: #eef;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+}
+
+.palette {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  width: 40px;
+  padding: 15px;
+  background: #2A2F36;
+  border-right: 1px solid #222;
+}
+
+.canvas {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 71px;
```

Comment:
Not super important for the demo, but consider trying out [Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) for handling CSS layout. With Flexbox, you wouldn't need to hard code a left value for the canvas, but would let Flexbox stretch the canvas to fill the available space.